content/en/database_as_a_service.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Database as a Service (DBaaS)
+status: Completed
+category: Technology
+---
+
+## What it is
+
+Database-as-a-Service (DBaaS) is a service managed by a cloud operator (public or private) that supports applications without requiring the application team to perform traditional database administration functions. With DBaaS, application developers should not need to be database experts, nor should they need to hire a database administrator (DBA) to keep the database up to date.
+
+## Problem it addresses 
+
+Traditionally in on-premise setups, organizations have to invest in additional storage and processing capacity on a regular basis to accommodate database expansion. Developers also need to provision and configure database with the help of IT infrastructure team. Deployment speed and database driven applications takes longer time to load and execute. This lack of features and scalibilty leads to increased costs and slow applications.
+
+## How it helps
+
+With DBaaS, it is possible to manage diverse sets of database from a single platform. Once a database is deployed, the cloud based service provider, is in charge of all backend operations to keep it running smoothly, including configuration management, backups, patches, upgrades, service monitoring, and more, with a user-friendly interface to manage it all. This helps organizations to develop enterprise level applications faster and keep if affordable at same time.
```

Comment:
Here are some edits. As to the content, I'll need a technical reviewer to look at it as well. 

> DBaaS allows users to manage databases from a single platform, while the service provider handles all backend operations. The DBaaS provider ensures the database runs smoothly, takes care of all configuration management, backups, patches, upgrades, service monitoring, and provides a user-friendly interface. This helps organizations develop enterprise-grade applications faster while minimizing database costs.

Reply:
I don't quite get the single platform line and I'm not sure it adds a ton to the definition. The tl;dr on a DBaaS is you get the database API without needing to do most of the operations and management of a database itself.

Reply:
Overall the content looks amazing. Great work @Asmit2952!

Reply:
> Overall the content looks amazing. Great work @Asmit2952!

Thank you!!

Reply:
Thanks @Asmit, I agree the content looks good. I would remove the single platform thing like Jason suggested and also replace "backend" with "administration/administrative".

Reply:
Okay, I'll do that

Reply:
Do I completely remove single platform thing, because cloud platform looks fine

Reply:
Okay then, I'll remove that line